changelogs/unreleased/gh-10310-runtime-grant-lua-call-via-config.md

```diff
@@ -0,0 +1,5 @@
+## feature/box
```

Comment:
Nit: This is config's feature (from a user perspective). Let's use the `## feature/config` category.

Reply:
Replaced with `feature/config`